fighthealthinsurance/admin.py

```
@@ -42,6 +43,7 @@
     PatientUser,
     UserDomain,
     ProfessionalDomainRelation,
+    VerificationToken,
```

Ruff (0.12.2)

`fhi_users.models.VerificationToken` imported but unused

Remove unused import: `fhi_users.models.VerificationToken`

(F401)